packages/mobile/src/Themes/natds.ts

```diff
@@ -1,11 +1,11 @@
 import { themes } from '@naturacosmeticos/natds-styles';
```

Comment:
You changed natds theme name to natura on styles package. Change this file name to natura too.